internal/scheduler/submitcheck.go

```diff
@@ -56,10 +56,11 @@ func (srv *SubmitChecker) CheckApiJobs(jobs []*api.Job) (bool, string) {
 		if gangId == "" {
 			continue
 		}
-		reqs, err := PodRequirementsFromJobs(srv.priorityClasses, jobs)
-		if err != nil {
-			return false, fmt.Sprintf("error encountered extracting scheduling requirements")
-		}
+		reqs := PodRequirementsFromLegacySchedulerJobs(jobs, srv.priorityClasses)
+		// TODO: Fix
```

Comment:
does this need fixing/removing before merge?

Reply:
This function doesn't even return an error. I've removed the todo.